drivers/tee/optee/core.c

```diff
@@ -223,8 +223,10 @@ static void optee_get_version(struct tee_device *teedev,
 	};
 	struct optee *optee = tee_get_drvdata(teedev);
 
-	if (optee->sec_caps & OPTEE_SMC_SEC_CAP_DYNAMIC_SHM)
+	if (optee->sec_caps & OPTEE_SMC_SEC_CAP_DYNAMIC_SHM) {
 		v.gen_caps |= TEE_GEN_CAP_REG_MEM;
+		pr_info("Dynamic shared memory is enabled and used\n");
```

Comment:
"enabled" sounds good, but "used" depends on `tee-supplicant`...

Reply:
`and used` removed.